src/dash-adapter.ts

```diff
@@ -1312,13 +1320,37 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
         error = error.data[0];
         this._requestFilterError ? (this._requestFilterError = false) : (this._responseFilterError = false);
       }
+      if (error.code === 1002) {
+        error.severity = this._onError1002ChangeSeverity() ? Error.Severity.CRITICAL : error.severity;
+      }
       this._trigger(EventType.ERROR, new Error(error.severity, error.category, error.code, error.data));
       DashAdapter._logger.error(error);
+
       if (error.severity === Error.Severity.CRITICAL) {
         this.destroy();
       }
     }
   }
+  private _onError1002ChangeSeverity() {
+    const secondsErrorRepeat = 15
+    const errorsCounterToChangeSeverity = 3;
+    const getCurrentTimeInSeconds = (): number => {
+      return Date.now() / 1000;
+    };
+    const interval = getCurrentTimeInSeconds() - this._errorFirstOccuredTime
+    this._error1002Counter += 1;
+    if ((interval > secondsErrorRepeat) && (this._error1002Counter > errorsCounterToChangeSeverity)) {
+      this._errorFirstOccuredTime = 0
+      this._error1002Counter = 0;
```

Comment:
what does code 1002 stand for ? maybe a text description would be clearer than the number